packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSliderForm.component.js

```diff
@@ -72,40 +71,33 @@ const BadgeSliderForm = ({
 	const isErroneous = useMemo(() => getValidator(decimal, min, max), [decimal, min, max]);
 
 	useEffect(() => onChange(null, value), [onChange, value]);
-	const schema = {
-		autoFocus: true,
-		disabled: false,
-		type: 'number',
-		schema: {
-			minimum: min,
-			maximum: max,
-			step,
-		},
-	};
 
 	return (
 		<form className={theme('tc-badge-slider-form')} id={`${id}-slider`} onSubmit={onSubmit}>
-			<RichTooltip.RichLayout.Body
-				id={`${id}-badge-body`}
-				className={theme('tc-badge-slider-form-body')}
-			>
+			<Rich.Layout.Body id={`${id}-badge-body`} className={theme('tc-badge-slider-form-body')}>
 				<div className={theme('tc-badge-slider-form-body-row')}>
 					{icon && <Icon name={icon.name} className={theme('tc-badge-icon', icon.class)} />}
 					{editing ? (
-						<UIForm.fields.Text
+						<input
 							id={`${id}-input`}
-							onChange={(_, { value: v }) => {
+							autoFocus
+							className="form-control"
+							min={min}
+							max={max}
+							step={step}
+							onChange={event => {
+								const v = event.target.value;
 								setInput(v);
 								setValue(!isErroneous(v) ? v : defaultValue);
 							}}
-							onFinish={() => {
+							onBlur={() => {
 								setInput(value);
 								setValue(value);
 								setSlider(value);
 								setEditing(false);
 							}}
-							schema={schema}
-							value={input}
+							type="number"
+							value={value}
```

Comment:
why do you change value from {input} to {value} ?
I would tend to think it is input because this is what the user has typed